### PR TITLE
Translate 5 terminos a español

### PR DESCRIPTION
**Pull request by demar01**
## Author: 

- @demar01

## Language: 

-  Español

## Terms defined:

- "media"
- "R (programming language)"
- "Stack Overflow"
- "standard deviation"
- "YAML"

## Editor

Assign the PR based on the language to the following person:

| Language   | Person                       |
|:----------:|:----------------------------:|
| Afrikaans  | @jsteyn, @elletjies          |
| Arabic     | @BatoolMM                    |
| English    | @baileythegreen, @zkamvar    |
| French     | @fmichonneau                 |
| Japanese   | @tomkellygenetics, @masamiy  |
| Portuguese | @beatrizmilz                 |
| Spanish    | @ian-flores, @demar01                  |

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -6136,3 +6182,11 @@
       no lugar de parênteses e vírgulas usados em [JSON](#json). YAML é frequentemente usado em 
       arquivos de configuração e na definição de [parâmetros](#parameter) para vários estilos
       de documentos em [Markdown](#markdown).
+   es:
+    term: "YAML"
```
Comment (baileythegreen) [inline glossary.yml:6186]:
```suggestion
  es:
   term: "YAML"
```

Fixing indentation errors.

File: glossary.yml [hunk 2]
```diff
@@ -379,7 +384,12 @@
       Um método de desenvolvimento de software que enfatiza vários passos pequenos e
       feedback contínuo ao invés de planejamento de longo prazo. [Programação exploratória](#exploratory_programming)
       costuma ser ágil.
-
+ es:
+    term: "desenvolvimiento ágil"
+    def: >
```
Comment (baileythegreen) [inline glossary.yml:389]:
```suggestion
  es:
    term: "desenvolvimiento ágil"
    def: >
```

Fixing indentation errors.

**Review by baileythegreen — CHANGES_REQUESTED** [review]
I have only looked at the causes of the failed checks. Once you make these changes, they should pass. Others will have to approve the language in your entries.

File: glossary.yml (hunk 1 above)
Comment (demar01) [reply glossary.yml]: Thanks! All checks have passed now, right?

Comment (baileythegreen) [reply glossary.yml]: Yes, they have. I'll approve them, and @ian-flores can approve the language.

**Review by baileythegreen — APPROVED** [review]

**Comment by beatrizmilz** [comment]
I think this PR is still being reviewed. @demar01 I'll reopen it, ok? Muchas gracias!

**Comment by demar01** [comment]
> I think this PR is still being reviewed. @demar01 I'll reopen it, ok? Muchas gracias!

Si, gracias! :)

**Review by ian-flores — APPROVED** [review]

**Comment by ian-flores** [comment]
Hi folks, just reviewed and merged. Thanks for your patience with my delay in the review

**Comment by baileythegreen** [comment]
@ian-flores This merge has now introduced some duplicate definitions for two terms, detailed here: https://github.com/carpentries/glosario/pull/103#pullrequestreview-550108274

> This is failing the linter check because two terms, absolute error and agile development, have two entries for the Spanish definition.
> 
> Absolute error: lines 97 and 125
> Agile development: lines 380 and 398
> 
> Please remove the duplicate entries, taking into consideration which of the given definitions is more complete.

As it's causing other PRs to fail the linter checks, I'm going to revert this merge so that the duplicate definition can be resolved.

**Comment by elletjies** [comment]
@all-contributors please add @gvwilson for translation

**Comment by elletjies** [comment]
@all-contributors please add @demar01 for translation

**Pull request merged**